init.zsh

```diff
@@ -76,8 +76,16 @@ function pmodload {
   local -a locations
   local pmodule
   local pmodule_location
-  local pfunction_glob='^([_.]*|prompt_*_setup|README*|*~)(-.N:t)'
 
+  # For some reason, Termux fails to locate the "prompt" module when NO_CASE_GLOB
```

Comment:
For fairly obvious reasons I'm not a huge fan of this fix - it's a dirty hack designed to work around a setting that probably shouldn't be set in the first place... but it also shouldn't cause issues either. I don't know why this glob would cause any problems anyway...